.github/workflows/pull-request.yaml

```diff
@@ -22,10 +22,10 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - name: Install Golang 1.20
+    - name: Install Golang
       uses: actions/setup-go@v5
       with:
-        go-version: '1.20'
+        go-version-file: 'go.mod'
```

Comment:
This action `setup-go` needs to be executed after the below `actions/checkout`, because at this stage the code has not yet been checked out, so `go.mod` file does not exist.